src/EditorFeatures/Core/Options/BraceCompletionOptions.cs

```diff
@@ -6,9 +6,8 @@ namespace Microsoft.CodeAnalysis.Editor.Options
 {
     internal static class BraceCompletionOptions
     {
-        public const string FeatureName = "BraceCompletion";
-
+        // This is serialized by the Visual Studio-specific LanguageSettingsSerializer
         [ExportOption]
-        public static readonly PerLanguageOption<bool> EnableBraceCompletion = new PerLanguageOption<bool>(FeatureName, "EnableBraceCompletion", defaultValue: true);
+        public static readonly PerLanguageOption<bool> EnableBraceCompletion = new PerLanguageOption<bool>(nameof(BraceCompletionOptions), nameof(EnableBraceCompletion), defaultValue: true);
```

Comment:
Yes. FeatureNames were never used in a meaningful way by the end of this refactoring. If I had a time machine I'd say we could get rid of them, but I decided to just make them match the class name for sanity's sake.